pom.xml

```diff
@@ -234,6 +254,40 @@
           <autoReleaseAfterClose>true</autoReleaseAfterClose>
         </configuration>
       </plugin>
+
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-shade-plugin</artifactId>
+        <version>3.1.1</version>
```

Comment:
The most recent version is 3.2.4, any specific reason to use 3.1.1?

Reply:
This was probably just the most recent version when the original change was made. There are quite a few other plugins that probably have older versions. I'd rather leave it like this as we know it works, we can always have a separate PR to update various versions later.